css/index.css

```diff
@@ -25,6 +27,22 @@ a:hover {
   background-size: cover;
   box-sizing: border-box; }
 
+.hackclub-logo {
+  font-family: "Amaranth", sans-serif;
+}
```

Comment:
Perhaps a better approach would be to use Hack Club's SVG logo.

A white version is over at https://github.com/hackclub/meta/blob/master/logos/hack_club_full_white.svg.